docs/projects/open_project/about.md

```diff
@@ -12,14 +12,16 @@ SLU Open Project is a web-based work management tool that enables shared service
 
 - **Source Code:** [https://github.com/oss-slu/open-project/](https://github.com/oss-slu/open-project/) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/open-project/)
 - **Client** SLU Center for Additive Manufacturing
-- **Current Tech Lead:** Lakshmi Prasanna Mamillapalli [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/lmamillapalli786) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/lakshmiprasana784/)
+- **Tech Lead:** Jack Crane [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/jackcrane) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/jackcranee/)
```

Comment:
Our current template for project pages has `**Current Tech Lead**`, but I kinda like how this makes it cleaner for longer lived projects. If we switch this on yours, I will want to make that change on all the pages for consistency. I'm trying to decide if it's worth the effort. What do you think?